boards/arm/nrf9160_pca10090/board.h

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __INC_BOARD_H
+#define __INC_BOARD_H
+
+#include <soc.h>
+
+
+#endif /* __INC_BOARD_H */

```

Comment:
It is uncommon to have a board.h file.

```
sebo@mach:~/ncs/zephyr/boards/arm$ find . -name board.h
find . -name board.h
./efr32_slwstk6061a/board.h
./efm32wg_stk3800/board.h
./reel_board/board.h
./nrf52_adafruit_feather/board.h
./bbc_microbit/board.h
./efm32hg_slstk3400a/board.h
./cc2650_sensortag/board.h
./efm32pg_stk3402a/board.h
./mps2_an385/board.h

```

and unprecedented to include soc.h from it.

So we need to either remove this file or come up with an explanation for what it is doing here.

Reply:
I agree, this can be removed and we should include `soc.h` directly instead.

Reply:
Yes - will do this, thanks.

Reply:
Removed.